projects/packages/post-list/src/class-post-list.php

```diff
@@ -32,12 +32,12 @@ public static function get_instance() {
 	}
 
 	/**
-	 * Sets up Post List action callbacks if needed.
+	 * Sets up Post List action callbacks.
 	 */
 	public function register() {
 		if ( ! did_action( 'jetpack_on_posts_list_init' ) ) {
 			add_action( 'admin_enqueue_scripts', array( $this, 'enqueue_scripts' ) );
```

Comment:
Should we move this to `add_thumbnail_scripts_and_filters`?

Reply:
I think that's worth taking a look at next iteration. 👍